site/class-1-load/youmghar/index.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<meta charset=utf8>
+<title>@youmghar</title>
+<meta content=width=device-width,initial-scale=1 name=viewport>
+<script src="https://d3js.org/d3.v4.min.js"></script>
+
+<h1>Dinners</h1>
+
+
+<script>
+ 
+  d3.csv('./index.csv', function (err, questions) {
+  if (err) throw err;
+  questions.forEach(function (q) {
```

Comment:
Tip: copying code is fine, but you should rename your variables in the future — `questions` don’t really work here!